hexagon_vm.h

```diff
@@ -1,70 +1,71 @@
 /*
```

Comment:
Can clang-format also format the minivm.S?

Reply:
Unfortunately: no.